.gitignore

```diff
@@ -125,3 +125,13 @@ dmypy.json
 
 .idea/
 .DS_Store
+
+# Submodule
+taplt/media_viewing_widgets
```

Comment:
I would recommend to keep the linked git submodule to link it directly in GitHub. Like this
![grafik](https://github.com/cgtuebingen/MedicalAnnotationFramework/assets/44415652/6bb83e88-c3da-43c5-8626-45a9abc5f1b4)